server/views/temporary-accommodation/components/lost-bed-info/macro.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
```

Comment:
For all the templates, add a newline at the end, and suggest add some more whitespace for readability